fastlane/Fastfile

```diff
@@ -0,0 +1,25 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
```

Comment:
I'm not sure if this is correct or not (I don't have experience with fastlane, but shouldn't this be `mac`, not `ios`?

https://docs.fastlane.tools/actions/default_platform/